force-app/main/default/classes/NKS_ChatReportDataBatch.cls

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Batchen skal ikke schedules, eller?

Reply:
Nei, dette er bare en engans batch for å gå igjennom alle de tidligere Chattene, derfor jeg ikke har lagd test og, siden den bare skal slettes etterpå 😄